### PR TITLE
Use `TouchableNativeFeedback ` for tab on Android

### LINKED ISSUE #125: [Android] Floating effect on tab tap

### PR DESCRIPTION
**Pull request by cancan101**
Closes #125

### CODE REVIEW

**Comment by chirag04** [comment]
cc @skv-headless

File: DefaultTabBar.js [hunk 1]
```diff
@@ -45,20 +47,41 @@ var DefaultTabBar = React.createClass({
...
+          <View>
+            { innerText }
+          </View>
+        </TouchableOpacity>
+      );
+    } else {
+      return (
+        <TouchableNativeFeedback
+          background={TouchableNativeFeedback.SelectableBackground()}
+          key={name}
```
Comment (skv-headless) [inline DefaultTabBar.js]: Or even better. Make component(wrapper). For android for example
```
<TouchableNativeFeedback {...this.props}>
{this.props.children}
</TouchableNativeFeedback>
```

**Comment by skv-headless** [comment]
If you fix my notes it would be great. Otherwise I can fix by myself.

File: DefaultTabBar.js (hunk 1 above)
Comment (cancan101) [inline DefaultTabBar.js]: I don't follow what you are suggesting spreading on Android.

File: DefaultTabBar.js [hunk 2]
```diff
@@ -45,20 +47,41 @@ var DefaultTabBar = React.createClass({
...
+        fontWeight: isTabActive ? 'bold' : 'normal'}}>{name}</Text>
     );
+
+    if (Platform.OS !== 'android') {
+      return (
+        <TouchableOpacity
+          key={name}
+          accessible={true}
+          accessibilityLabel={name}
+          accessibilityTraits='button'
```
Comment (cancan101) [inline DefaultTabBar.js]: https://facebook.github.io/react-native/docs/view.html#accessibilitylabel is not listed as iOS only.

**Comment by cancan101** [comment]
@skv-headless I made the one change that made sense, not sure about the others. LMK if good and then I will squash.

File: DefaultTabBar.js (hunk 2 above)
Comment (chirag04) [inline DefaultTabBar.js]: https://github.com/facebook/react-native/blob/3e760f48c5a28b592cb60c4060ca28a1574f6274/ReactAndroid/src/main/java/com/facebook/react/uimanager/BaseViewManager.java#L34

**Comment by skv-headless** [comment]
@cancan101 squash please and I'll merge.

**Comment by cancan101** [comment]
@skv-headless done

**Comment by skv-headless** [comment]
Thanks :+1:

**Pull request merged**